core/templates/components/forms/custom-forms-directives/edit-thumbnail-modal.template.html

```diff
@@ -50,7 +50,7 @@ <h3>Upload Thumbnail</h3>
             <i class="material-icons oppia-vcenter">&#xE14C;</i>
           </button>
           <div>
-            <img ng-if="uploadedImage" class="oppia-thumbnail-image" ng-src="<[uploadedImage]>" height="<[aspectRatio === '4:3' ? '186px' : '180px']>" width="<[aspectRatio === '4:3' ? '248px' : '320px']>">
```

Comment:
@DubeySandeep @kevintab95 the styling for the class `oppia-thumbnail-image` doesn't exist in the codebase. No mention of it in any css selector. The other class which is used in this context is `img-thumbnail`, that class was found in third party css. Is this expected?

Reply:
Yep we don't use it for styling but looks like `oppia-thumbnail-image` is used as a selector in `edit-thumbnail-modal.controller.ts`

Reply:
But looks like this PR has removed that so we don't really use `oppia-thumbnail-image` any more.

Reply:
@srijanreddy98 is this comment resolved? (I think we should remove the classes attribute or remove the `oppia-thumbnail-image`.)